Add limitation notes for create shared channel (#19509)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/channel-post.md

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Create a new [channel](../resources/channel.md) in a team, as specified in the request body.

> > **Note**: While creating a channel, only one members can be specified for shared channel however maximum of 200 members can be specified for private channel.

Reviewer: ```suggestion
> **Note**: While creating a channel, you can only add one member per shared channel; however, you can add up to 200 members per private channel.
```

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Create a new [channel](../resources/channel.md) in a team, as specified in the request body.
> **Note**: You can add a maximum of 200 members when you create a private channel.

> > **Note**: Specifying more than one member while creating the shared channel is not supported.

Reviewer: @devjha-ms ,  there isn't technically a rule against having multiple notes in a section, but we would want to avoid that because having two notes like that will dilute the stand-out factor of the notes.
I have a couple of options for you: 
We could combine them in one single note.
Move the notes to different locations so they're separate. 
Convert one of the notes to normal text. 
Also, is the second note a Known issue or just the default behavior? We could turn it into the standard text if it's the default behavior.
